src/statisticBlock.js

```diff
@@ -111,13 +109,35 @@ class StatisticBlock extends Component {
       let clientWidth = this.state.clientWidth;
       let clientHeight = this.state.clientHeight;
       let childHeight = 0;
+      let childWidth = 0;
+      let containerElm = this.refs.parent;
+      let childrenElm =containerElm.children;
+      let childrenCombinedWidth = 0;
+      let childrenHeight = childrenElm[0].clientHeight;
+      for (let e of childrenElm) {
+        childrenCombinedWidth = childrenCombinedWidth + e.clientWidth;
+      }
 
-      if(element.clientHeight == element.scrollHeight
-      && this.state.size == this.props.options.size
-      && !this.state.overflow) return;
-
+      // if(element.clientHeight == element.scrollHeight
+      // && this.state.size == this.props.options.size
+      // && !this.state.overflow) return;
+
+      if (elementClientWidth < childrenCombinedWidth || elementClientHeight < childrenHeight){
```

Comment:
Wouldn't elementclientwidth and childrencombinedwidth be roughly the same, excluding potentially padding/margin/border of the container?

Reply:
the should be roughly the same,, but the original problem was that the children together (childrencombinedwidth) get more space the the parent (elementclientwidth)